contracts/GnosisSafe.sol

```diff
@@ -38,6 +38,13 @@ contract GnosisSafe
     //);
     bytes32 private constant SAFE_MSG_TYPEHASH = 0x60b3cbf8b4a223d68d641b3b6ddf9a298e7f33710cf3d3a9d1146b5a6150fbca;
 
+    event SafeSetup(
```

Comment:
I miss `address indexed safeAddress`

Reply:
The safe address is the address that emits the event, so you can by default filter by it ;)

Reply:
You are right, and it should be the same to filter using the address as an indexed topic than this

Reply:
So no change needed, right? :D